src/target/efm32.c

```diff
@@ -620,57 +583,55 @@ bool efm32_probe(target *t)
 	}
 
 	/* Read the part family, and reject if unknown */
-	size_t device_index  = efm32_lookup_device_index(t, di_version);
-	if (device_index > (127-32)) {
+	size_t device_index = efm32_lookup_device_index(t, di_version);
+	if (device_index > (127 - 32)) {
 		/* unknown device family */
 		return false;
 	}
-	efm32_device_t const* device = &efm32_devices[device_index];
+	efm32_device_t const *device = &efm32_devices[device_index];
 	if (device == NULL) {
 		return false;
 	}
 
 	uint16_t part_number = efm32_read_part_number(t, di_version);
 
 	/* Read memory sizes, convert to bytes */
-	uint16_t flash_kib  = efm32_read_flash_size(t, di_version);
+	uint16_t flash_kib = efm32_read_flash_size(t, di_version);
 	uint32_t flash_size = flash_kib * 0x400;
```

Comment:
U on the end of constants in an unsigned context - don't want to accidentally invoke integer promotion conversions here.